--- a/qtwebengine/src/3rdparty/chromium/build/mac/find_sdk.py	2020-05-31 10:04:46.000000000 +0100
+++ b/qtwebengine/src/3rdparty/chromium/build/mac/find_sdk.py	2020-05-31 10:05:38.000000000 +0100
@@ -63,7 +63,7 @@
   options, args = parser.parse_args()
   if len(args) != 1:
     parser.error('Please specify a minimum SDK version')
-  min_sdk_version = args[0]
+  min_sdk_version = '.'.join(str(i) for i in parse_version(args[0])[:2])
 
 
   # If we are passed the developer dir, then we don't need to call xcode-select.
